Rb.Forms.Barcode.nuspec

```diff
@@ -3,42 +3,33 @@
   <metadata>
     <id>Rb.Forms.Barcode</id>
     <title>Barcode scanner plugin for Xamarin.Forms</title>
-    <version>0.4.2-beta</version>
+    <version>0.5.0-beta</version>
     <authors>Ota Mares</authors>
     <owners>reBuy reCommerce GmbH</owners>
     <licenseUrl>https://github.com/rebuy-de/rb-forms-barcode/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/rebuy-de/rb-forms-barcode</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes. Build upon the decoding capabilities of ZXing.Net.Mobile, full of awesomeness and also unicorns.
-Please check the website for further details.
+    <description>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes full of awesomeness and also unicorns. Please check the website for further details.
 
 Beta notice: Currently only android is supported.</description>
     <summary>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes.</summary>
-    <releaseNotes>0.4.2-beta:
```

Comment:
Die Infos wären als release-Tag echt schön. Habe ich beim letzten Mal aber auch vergessen. ist mir erst jetzt in den Sinn gekommen. Soll ich das noch nachtragen?